### DISCUSSION TITLE
Why cant i use the option "force https"

### DISCUSSION THREAD

**Discussion by wegGehamstert** [AzuraCast/AzuraCast#4197] [Q&A]
I dont know why this is like that but i set up Lets Encrypt and all the stuff but my site dont want to use https://.
It looks like this:

![image](https://user-images.githubusercontent.com/78315888/119217692-1a322f00-badc-11eb-85cc-dc1d91366a80.png)

I cant click on the box

**Comment by Psychoses** [comment]
maybe https://github.com/AzuraCast/AzuraCast/issues/4115#issuecomment-831670066
